front-end/src/pages/FollowListPage.tsx

```diff
@@ -5,6 +5,7 @@ import { Row, Col, Card, CardBody, CardTitle, Container, Button, CardText, Nav,
 import AuthorListItem from '../components/AuthorListItem';
 import { Author } from '../types/Author';
 import { UserLogin } from '../types/UserLogin';
+var classNames = require('classnames');
```

Comment:
doing this instead of installing `classnames` package because it kept breaking the frontend. maybe someone else can fix it? idk

Reply:
What's the error with this? I'd be cautious to do something like this since I don't know how webpack handles these `requires`

Reply:
I read up what `classnames` does. If you really want to, you can use `clsx` instead to use the import statement: https://www.npmjs.com/package/clsx. Otherwise, this is probably fine, since I suspect `classnames` uses pre ES6 import/exports

Reply:
I will take a look at using that!